memtest86plus: new, 7.00

rickliu2000 opened the pull request:
Topic Description
-----------------

- memtest86plus: apply general fix-ups
    - Backport pre-upstream LoongArch support patches from
    https://github.com/kilaterlee/memtest86plus/tree/SubmitLoongArch.
    - Adapt build and grub.d scripts to detect LoongArch.
    - Adapt to i486 (Afterglow).
    - Add header to reference original Arch Linux script.
- memtest86plus: style linting
- memtest86plus: new, 7.00

Package(s) Affected
-------------------

- memtest86plus: 7.00

Security Update?
----------------

No

Build Order
-----------

```
#buildit memtest86plus
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] LoongArch 64-bit `loongarch64`

rickliu2000 committed:
memtest86plus: new, 7.00

MingcongBai reviewed (changes requested):
Please review the Styling Manual and lint the build scripts.

rickliu2000 committed:
memtest86plus: style linting

MingcongBai committed:
memtest86plus: apply general fix-ups

- Backport pre-upstream LoongArch support patches from
  https://github.com/kilaterlee/memtest86plus/tree/SubmitLoongArch.
- Adapt build and grub.d scripts to detect LoongArch.
- Adapt to i486 (Afterglow).
- Add header to reference original Arch Linux script.

MingcongBai committed:
memtest86plus: bump REL for topic Revision Marking Guidelines

MingcongBai committed:
memtest86plus: fix build scripting

MingcongBai committed:
memtest86plus: refresh patchset

rickliu2000 commented:
![image](https://github.com/user-attachments/assets/7043f44e-6a7d-4517-b687-b432e2d778e3)
memtest86plus tested on LoongArch

rickliu2000 committed:
memtest86plus: fix grub script syntax

MingcongBai reviewed (approved):
LGTM on both AMD64, LoongArch.
